flocx_ui/static/dashboard/project/flocx/contract-list/contract-list.html

```diff
@@ -0,0 +1,43 @@
+<div ng-controller="ContractListController as ctrl">
+  <div class="col-lg-10">
+    <h3 style="margin: 24px 0;">Contracts</h3>
+    <div class="btn-group" role="group">
+      <button
+        type="button"
+        class="btn"
+        ng-class="{ 'btn-default': ctrl.pane !== 'active', 'btn-primary': ctrl.pane === 'active' }"
+        ng-click="ctrl.showPane('active')"
+      >
+        Active
+      </button>
+      <button
+        type="button"
+        class="btn"
+        ng-class="{ 'btn-default': ctrl.pane !== 'expired', 'btn-primary': ctrl.pane === 'expired' }"
+        ng-click="ctrl.showPane('expired')"
+      >
+        Expired
+      </button>
+    </div>
+    <table class="table" style="margin-top: 16px;">
+      <thead>
+        <tr>
+          <th scope="col">Contract ID</th>
```

Comment:
What do you think about this idea? By clicking on a contract, the owner of the node will be able to see the offer that was made and the user will be able to see the bid that was made. Of course, this is a feature that we have to worry about later once we have the basic system in place. But we have to provide a mechanism to show the offer or the bid that is tied to the contract. We might track this as an issue if you feel this makes sense.

Reply:
I like this idea, but I believe that there are more important user stories to complete before this.

Reply:
Yes. We will have to worry about it only when all the user stories that we already have onboard have been addressed.